nuxeo-core/nuxeo-core-query/src/main/java/org/nuxeo/ecm/core/query/sql/model/WhereClause.java

```diff
@@ -28,13 +28,13 @@
 
     private static final long serialVersionUID = -6192874148042567401L;
 
-    public final Predicate predicate;
+    public final Expression predicate;
```

Comment:
It would be clearer if the variable is called `expression`.

Reply:
Agreed but I wanted to minimize the diff